events/event/migrations/0002_auto_20161020_1027.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-# Generated by Django 1.10.2 on 2016-10-20 07:27
-from __future__ import unicode_literals
```

Comment:
Why do you need to delete migration? new migration should be created if changes made.